### PR TITLE
#506 upgrade to net 5

### PR DESCRIPTION
**Pull request by brano98**
Afterall it was quite easy to migrate to .NET 5. Just few changes but I left 3-4 warning where obselete things are used. Those can be refactored in next migration or separate task.

### CODE REVIEW

File: WebAPI/WebAPI.csproj [hunk 1]
```diff
@@ -20,6 +21,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.5" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.9" />
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.11.1" />
```
Comment (CholujMartin) [inline WebAPI/WebAPI.csproj:24]: Why Azure added ?

**Review by CholujMartin — APPROVED** [review]
Looks good and if it works good job, but maybe wait for my own logind student id missing merge into develop and then try to merge this with develop because of possible conflicts.

File: WebAPI/WebAPI.csproj (hunk 1 above)
Comment (brano98) [reply WebAPI/WebAPI.csproj:24]: I needed to test dockerfile and vs automatically added this I guess....

**Comment by brano98** [comment]
Ok, I it can wait ... but I am preparing docker integration to visual studio and it is already based on this branch so it will block my branch then.

**Pull request merged**